.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "terminal.integrated.fontFamily": "Hack Nerd Font"
+}
```

Comment:
`.vscode` should probably be added to the list on `.gitignore`. I'm assuming this is local visual studio configs so not directly related to the project.

Reply:
On it...